Add `Skinning Contest: Unchained - Voting Open` news post (#11641)
ppy/osu-wiki: news/2024/2024-06-23-skinning-contest-unchained-voting.md

---
layout: post
title: "Skinning Contest: Unchained - Voting Open"
date: 2024-06-23 12:00:00 +0000
---

The skinning community let their creativity run wild, and now it's time to cast your votes and decide upon your favourites!

![](/wiki/shared/news/2024-06-23-skinning-contest-unchained-voting/banner.jpg)

The submission phase has officially ended on May 24th, and after a lot of hours put in by the skinship team in order to validate and screen all the entries, it's finally time to move on to the voting phase.

> As mentioned in the [announcement news post](https://osu.ppy.sh/home/news/2024-04-09-skinning-contest-unchained), there aren't any theme restrictions for this contest, which has translated into **27** brand-new skins, all yearning for your attention! As always we highly recommend to head over to the [contest listing on the completed skins compendium](https://compendium.skinship.xyz/contests/5) and check out all the entries yourself, but if you are not into collecting skins the team has compiled a playlist of showcase videos for your viewing pleasure:

Reviewer: ```suggestion
As mentioned in the [initial announcement](https://osu.ppy.sh/home/news/2024-04-09-skinning-contest-unchained), there aren't any theme restrictions for this contest, which has translated into **27** brand-new skins, all yearning for your attention! As always, we highly recommend to head over to the [contest listing on the completed skins compendium](https://compendium.skinship.xyz/contests/5) and check out all the entries yourself, but if you are not into collecting skins, the team has compiled a playlist of showcase videos to help you still pick your favourites:
```

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/videoseries?list=PLA2Nwtxa4WMqNujKzed748MxnJ0BfZ6YY" frameborder="0" allowfullscreen></iframe>
</div>

> Made up your mind yet? If so, head over to the [contest page](https://osu.ppy.sh/community/contests/208) and *unleash* your **3 votes**! The voting phase will last for the next **14 days**, so if you haven't decided yet, there is plenty of time to do so.

Reviewer: ```suggestion
Made up your mind yet? If so, head over to the [contest page](https://osu.ppy.sh/community/contests/208) and *unleash* your **3 votes**! The voting phase will last for the next **14 days**, so there will be plenty of time to decide.
```
